README.md

```diff
@@ -14,7 +14,7 @@ val httpService = new RhoService {
 }
 ```
 
-See the [wiki](https://github.com/http4s/rho/wiki) and the [tests](https://github.com/http4s/rho/blob/master/core/src/test/scala/ApiExamples.scala) for more examples.
+See the [tutorial](https://github.com/eklavya/rho/blob/master/Rho.md), [wiki](https://github.com/http4s/rho/wiki) and the [tests](https://github.com/http4s/rho/blob/master/core/src/test/scala/ApiExamples.scala) for more examples.
```

Comment:
Should this point to your fork?

Reply:
👍

Reply:
Easy thing to have happen by accident. 👍

Reply:
yup, just me being silly :)